### PR TITLE
Updated manifest

### PR DESCRIPTION
**Pull request by MORITZ0405**
## Type of change

- [ ] Add new directory for a server
- [X] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/tornadomc/manifest.json [hunk 1]
```diff
@@ -9,10 +9,10 @@
     "de"
   ],
   "social": {
-    "web": "https://forum.tornadomc.net",
+    "web": "https://tornadomc.net",
     "web_shop": "https://store.tornadomc.net",
     "tiktok": "@tornadomc.net",
-    "discord": "https://discord.gg/55PZcudMHE",
+    "discord": "https://discord.gg/857mVgfKXZ",
```
Comment (DoJapHD) [inline minecraft_servers/tornadomc/manifest.json:15]: Doesn't seem like Discord for server?
https://i.imgur.com/JcHE68p.png

File: minecraft_servers/tornadomc/manifest.json [hunk 2]
```diff
@@ -9,10 +9,10 @@
     "de"
   ],
   "social": {
-    "web": "https://forum.tornadomc.net",
+    "web": "https://tornadomc.net",
```
Comment (DoJapHD) [inline minecraft_servers/tornadomc/manifest.json:12]: Does not load
https://i.imgur.com/b1WrTi2.png

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/tornadomc/manifest.json (hunk 2 above)
Comment (MORITZ0405) [reply minecraft_servers/tornadomc/manifest.json:12]: Fixed

File: minecraft_servers/tornadomc/manifest.json (hunk 1 above)
Comment (MORITZ0405) [reply minecraft_servers/tornadomc/manifest.json]: Fixed

File: minecraft_servers/tornadomc/manifest.json (hunk 2 above)
Comment (DoJapHD) [reply minecraft_servers/tornadomc/manifest.json:12]: Still not loading

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**